libs/core/tsconfig.lib.json

```diff
@@ -15,7 +15,8 @@
     "lib": [
       "dom",
       "es2015",
-      "ESNext.AsyncIterable"
+      "ESNext.AsyncIterable",
+      "es2018.regexp"
```

Comment:
What does this do?

Reply:
Using regex breaks without it https://github.com/microsoft/TypeScript/issues/29465#issuecomment-455489648